src/Platform/Http/Middleware/Access.php

```diff
@@ -56,7 +56,7 @@ public function handle(Request $request, Closure $next, string $permission = 'pl
 
         // The current user is already signed in.
         // It means that he does not have the privileges to view.
-        abort(403);
+        abort(config('platform.status', 403));
```

Comment:
Wouldn't it be enough to specify the rule only here? After all, if you want to hide the very presence of limited technologies for example for bots, then this should weed them out.

Reply:
There can be a case when you want to hide sections of platform from users with lower access level